inference_server/utils/requests.py

```diff
@@ -92,35 +85,35 @@ def parse_field(kwargs: dict, field: str, dtype: int, default_value: Any = None)
         return default_value
```

Comment:
nit: In the `parse_field` func above, the type hit for `dtype` should be `type` rather than `int`

Reply:
Thanks for catching this